src/components/NoteRoot/NoteRoot.js

```diff
@@ -68,10 +69,14 @@ class NoteRoot extends React.Component {
 
   render() {
     const { annotation, renderContents, isEditing, closeEditing, searchInput, contents } = this.props;
+    const annotationState = annotation.getStatus();
 
     return (
       <div className="NoteRoot">
         {this.renderHeader()}
+        {annotationState && annotationState !== '' && annotationState !== 'None' &&
```

Comment:
I think if `annotationState` is truthy then it implies it won't equal to `''` so we can just have 

```javascript
annotationState && annotationState !== 'None' && ...
```

Reply:
Resolved in dcb321c. Thank you!